Felienne/hedy
# [UI IMPROVEMENT] Differentiates between keyword and non-keyword languages on menubar

**Description**
In this PR we more clearly differentiate between supported and non-supported keyword languages of Hedy. Instead of one big language selector we divide them into two rows clearly differentiating the support. This can still be a bit confusing as some keyword supported languages are not translated at all on other aspects, but this should automatically be fixed as more translations are contributed. All is done dynamically and nothing has to be updated when a new (keyword)language is added.

**Fixes**
This PR fixes #2618.

**How to test**
Navigate to the menubar and notice then new structure of the language selector. Notice that this selector is only visible when not being logged in. Verify that alle features such as filtering and selecting are still working as expected.

![Schermafbeelding 2022-05-02 om 08 38 27](https://user-images.githubusercontent.com/38155424/166194697-d9d4180b-c584-4051-b46d-1c1dda05371c.png)
